README.md

```diff
@@ -43,6 +43,8 @@ The code that enables token auth should be placed in `vcl_recv`. This is an exam
   unset req.http.X-Exp;
 ```
 
+> NOTE: Please generate your own key before using this code.
```

Comment:
Nitpick: Could you add a mention about _why_ you need to do this? I think its important to justify the advice by providing an understanding of what would happen if it isn't followed.